### ISSUE TITLE
Add Content: Conda and Windows Installation

### ISSUE THREAD

**Issue by jroberayalas** [koaning/calmcode-feedback#72]
Sorry, just a quick question: what's the difference between creating a virtual environment with virtualenv VS conda? And which one is more recommended?

**Comment by koaning** [comment]
Thats a good question. I'm not *that* familiar with conda. My understanding is that conda includes a virtualenv but can also handle dependencies outside of python like R-environments and system requirements. If I recall correctly you should be able to just `pip install` from a conda environment but I could be missing a detail.

**Comment by cipri-tom** [comment]
Indeed, `conda` has the ability to create separated environments, such that _stuff_ installed with `conda install` can be separated by project. I say _stuff_ exactly because conda can handle a lot of different things outside of python. For example, it can install CUDA, or various C++ binaries (ogr2ogr), which simply cannot be installed by `pip`.

You _can_, indeed, use `pip install` inside a conda environment, if that environment includes `python`, and the packages will end up in _that_ `python`'s `site-packages`. So it kinda works like a regular `virtualenv`. However, note that python packages can also be installed via `conda` (e.g. `conda install pandas`), and this will **not** take into account the packages installed by `pip` -> if you combine `pip install` and `conda install`, you should know what you're doing™️.

Since `conda` can install almost anything, it can also install `python`. So I prefer to use `conda` to manage my environments because it allows me to specify which python version is needed for that environment, whereas with `virtualenv` you're bound to the python version used to create the it.

I don't think there is a recommended way. I think for most projects you don't need complicated external dependencies (CUDA), so you can use just `virtualenv`. As soon as you need `conda` (e.g. PyTorch), it is good to know what it does and how it works.

I always use conda to manage my environments. Since I work on libraries, I use `pip install` for library dependencies, which go in a `requirements.txt` in the library repository. And I use `conda install` for "development" dependencies (jupyter, ipython), since these are invisible to `pip`, therefore I can always do `pip freeze > requirements.txt` without worrying that consumers of my little library will end up installing `jupyter`

**Comment by jroberayalas** [comment]
Thanks @cipri-tom. Your reply was very helpful!

**Comment by koaning** [comment]
I'll consider the question answered but I will keep this issue around. I think a video on conda (and installation on windows) might be very useful for the platform. I'll change the name to reflect this. 

@cipri-tom thanks for the elaborate answer 👍
